src/Ionide.ProjInfo/Types.fs

```diff
@@ -68,6 +68,13 @@ module Types =
         Items: ProjectItem list
         Properties: Property list
         CustomProperties: Property list
+        /// This contains all Project Properties seen during evaluation
+        AllProperties: Map<string, Set<string>>
+        /// This contains all Project Items seen during evaluation with their values and associated metadata
+        AllItems: Map<string, Set<string * Map<string, string>>>
+        /// Will have Key Value pairs like "PkgIonide_Analyzers" -> "C:\Users\username\.nuget\packages\ionide.analyzers\0.14.7"
+        /// The "analyzers/dotnet/fs" subfolder is not included here, just the package root
+        Analyzers: Map<string, Set<string>>
```

Comment:
Could make this type a bit more robust with a path to the analyzer dll + the root folder.